.github/workflows/cypress-workflow.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - development-*
```

Comment:
This was added to run the integration tests and unit tests for dev branch. We could remove this when merging to main.